Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:18-bullseye
+# From https://github.com/pnpm/pnpm/issues/4837
+
+EXPOSE 8890/tcp
+EXPOSE 8891/udp
+
+COPY ./package.json .
```

Comment:
Would it make sense to copy the whole application first and only then do the install sequence? Also I'd prefer if noray had its own directory in the container.